src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/dto/OverseasEntitySubmissionDto.java

```diff
@@ -8,16 +8,19 @@
 
 public class OverseasEntitySubmissionDto {
 
+    public static final String ENTITY_FIELD = "entity";
```

Comment:
I defined these as the exact JSON name string is also used in the `FilingsService` class and I didn't like the duplication. What do we think?

Also, in an agile fashion, I've only defined constants for the fields currently going to CHIPS but could define them all now if that's the preference.

Reply:
seems ok to me, the filing service will just reference the constant in the dto to ensure the json fields are named the same when exporting from /filings endpoint. The json properties naming will remain unchanged, just referencing a constant to avoid typos etc..  Happy to convert them as we go.